components/Form.js

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+export default function Form() {
+  const [title, setTitle] = useState('');
+  const [lyrics, setLyrics] = useState('');
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    setTitle(event.target.title.value.trim());
+    setLyrics(event.target.lyrics.value.trim());
+  }
+
+  return (
+    <>
+      <OutputContainer>
+        <Output>"{title ? title : ''}"</Output>
+        <Output>{lyrics ? lyrics : ''}</Output>
+      </OutputContainer>
+      <InputForm onSubmit={handleSubmit}>
+        Start creating your first idea:
+        <div>
+          <label for="title">Title: </label>
+          <input type="text" id="title" name="title" required />
+        </div>
+        <div>
+          <LyricsField
+            type="text"
+            placeholder="Keep your ideas rolling!"
+            id="lyrics"
+            name="lyrics"
+            required
+          />
+        </div>
+        <Button type="submit">You rock!</Button>
```

Comment:
The button text could be a bit more descriptive. Maybe people wouldn't recegnize it as a submit button :)